jni/cpp/TWJNIString.cpp

```diff
@@ -9,7 +9,9 @@
 #include "TWJNIString.h"
 
 jstring _Nonnull TWStringJString(TWString *_Nonnull string, JNIEnv *env) {
-    return env->NewStringUTF(TWStringUTF8Bytes(string));
+    jstring result = env->NewStringUTF(TWStringUTF8Bytes(string));
+    TWStringDelete(string);
+    return result;
```

Comment:
Check that result is freed.

Reply:
I think result is managed by Java GC, @Defuera will run the profile program again

Reply:
will create a new branch for JNI fix, looks like we have to change codegen